images/prow-tests/Dockerfile

```diff
@@ -29,10 +29,12 @@ RUN docker-credential-gcr configure-docker
 RUN apt-get install -y uuid-runtime  # for uuidgen
 RUN apt-get install -y rubygems  # for mdl
 
-# Update go version to 1.12rc1
-RUN go get golang.org/dl/go1.12rc1
-RUN go1.12rc1 download
-RUN cp $(which go1.12rc1) $(which go)
+# Install go at 1.12
```

Comment:
Thanks for the update, Chao. To avoid future questions, concerns or "smater updates" here, I suggest you add a comment here that this is the way it's done for kubekins, and a permalink to the original code:

https://github.com/kubernetes/test-infra/blob/1e9b5dc3de4b268aab57c9c48120aa3dcf096bc6/images/kubekins-e2e/Dockerfile#L64

Reply:
Sounds good, added